### PR TITLE
feat(IPVC-2246): add seqrepo load to uta build script

### PR DESCRIPTION
**Pull request by bsgiles73**
1.  Add instructions on setting up SeqRepo docker volume
2. Add steps to run-uta-build.sh script to load novel sequences
3. Add UTA database diff to build script

Example output of seqrepo load:

```
...
+ seqrepo load -n NCBI -i 2024-02-20 /temp/refseq/H_sapiens/mRNA_Prot/human.test.rna.fna.gz /temp/refseq/H_sapiens/mRNA_Prot/human.test.protein.faa.gz
human.test.protein.faa.gz: 100%|██████████| 2/2 [00:03<00:00,  1.75s/file]
...
```

Example output of uta_diff

```
+-----------------------+------+---------+---------+-----+---------+------+------------------------------------------------+
|         table         |  t   |    n1   |    n2   | nu1 |    nc   | nu2  |                      cols                      |
+-----------------------+------+---------+---------+-----+---------+------+------------------------------------------------+
| associated_accessions | 7.0  |  265035 |  265035 |  0  |  265035 |  0   |              tx_ac,pro_ac,origin               |
|          exon         | 41.5 | 8310936 | 8312219 |  0  | 8310936 | 1283 |                       *                        |
|        exon_aln       | 37.9 | 5604190 | 5604190 |  0  | 5604190 |  0   | exon_aln_id,tx_exon_id,alt_exon_id,cigar,added |
|        exon_set       | 6.7  |  894082 |  894235 |  0  |  894082 | 153  |                       *                        |
|          gene         | 0.3  |  45857  |  45871  |  0  |  45857  |  14  |                      hgnc                      |
|          meta         | 0.0  |    4    |    4    |  0  |    4    |  0   |                       *                        |
|         origin        | 0.0  |    6    |    6    |  0  |    6    |  0   |                       *                        |
|          seq          | 23.0 |  340384 |  340384 |  0  |  340384 |  0   |                       *                        |
|        seq_anno       | 2.3  |  360063 |  360063 |  0  |  360063 |  0   |                       *                        |
|       transcript      | 10.9 |  314227 |  314380 |  0  |  314227 | 153  |                       ac                       |
+-----------------------+------+---------+---------+-----+---------+------+------------------------------------------------+
```

### CODE REVIEW

**Review by nvta1209 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -294,22 +294,58 @@ To run local build of UTA you can follow these steps...
...
 
-2. Setup local database follow the steps above (`Installing with Docker`) to pull image, 
-   start and populate. __NOTE__...it does take about 5 minutes for the local database is ready.
+2. Setup local Docker container running UTA Postgresql database follow the steps above 
+   (`Installing with Docker`) to pull image, start and populate. __NOTE__...it does take 
+   about 5 minutes for the local database is ready.
 
-3. Build the image
+3. Setup local Docker volume for SeqRepo. This can be done by running the following two
+   commands. __NOTE__...this step does require aourn ~13Gb of local disk space and takes
```
Comment (sptaylor) [inline README.md]: nit: typo around

File: README.md [hunk 2]
```diff
@@ -294,22 +294,58 @@ To run local build of UTA you can follow these steps...
 
 1. Pick a UTA version as the base (ex. "uta_20210129b")
 
-2. Setup local database follow the steps above (`Installing with Docker`) to pull image, 
-   start and populate. __NOTE__...it does take about 5 minutes for the local database is ready.
+2. Setup local Docker container running UTA Postgresql database follow the steps above
```
Comment (sptaylor) [inline README.md]: This reads funny, maybe change follow->by following?

**Review by sptaylor — APPROVED** [review]
lgtm!

File: README.md (hunk 2 above)
Comment (bsgiles73) [reply README.md]: Oh you are right. Ok reworded this section. Thanks @sptaylor

**Pull request merged**